MVC Flowsheet Documentation (#1430)
watertap-org/watertap: docs/technical_reference/flowsheets/mvc.rst

where it absorbs heat from the condensing steam and vaporizes, leaving behind concentrated brine. The generated vapor
is then directed to a compressor, which elevates its temperature and pressure. This compressed vapor is recycled back
to the evaporator, where it condenses and provides the necessary heat for evaporating the preheated feed. The hot
concentrated brine and distillate pass through heat exchangers to preheat the incoming feedwater before exiting the
system. The flowsheet relies on the following key assumptions:

   * supports steady-state only
   * property package(s) supporting liquid and vapors is provided

.. figure:: ../../_static/flowsheets/mvc.png
    :width: 600
    :align: center

    Figure 1. MVC flowsheet

Documentation for each of the WaterTAP unit models can be found below.
    * `Pressure Changer <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/pressure_changer.html>`_
    * `Evaporator <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/evaporator.html>`_
    * `Compressor <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/compressor.html>`_

>     * `Condenser <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/condenser.html>`_

Reviewer: Since there is no documentation for each component of MVC (compressor, evaporator, condenser), I suggest referring to the MVC unit model instead of linking to each component individually. The current links are non-existent and non-functional. The link to MVC unit model: 
https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html

Author: I've updated the compressor, evaporator and condenser links to point to the mvc unit model documentation, and updated the pressure changer to point to the IDAES documentation

    * `Translator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/translator.html>`_

Documentation for each of the property models can be found below.
    * `Water <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_
    * `Seawater <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_

Documentation for the costing relationships can be found below.
    * `WaterTAP Costing Package <https://watertap.readthedocs.io/en/latest/technical_reference/costing/watertap_costing.html>`_

The objective function is to minimize the levelized cost of water, which can be represented by the following equation
where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
:math:`f_{util}` represents the utilization factor:

    .. math::

        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}

Degrees of Freedom
------------------

> The following variables are specified for simulating the MVC flowsheet based on the default settings:

Reviewer: ```suggestion
The following variables are specified based on the default settings:
```

> The following variables are initially specified for simulating the MVC flowsheet (i.e., degrees of freedom = 0):

Reviewer: I don't believe we need to list all the variables fixed for initialization purposes (DOF=0). The main goal of this flowsheet is to find the optimum values for the evaporator area, brine and distillate heat exchanger areas, evaporator temperature, and compressor pressure ratio. I suggest removing the DOF=0, and the mentioned variables from the list

Reviewer: I have made suggestions accordingly

Author: Done

> The following variables are initially specified for simulating the MVC flowsheet (i.e., degrees of freedom = 0):

Reviewer: ```suggestion
The following variables are specified for the MVC flowsheet based on the default setting:
```

    * feed water conditions (mass flow, mass fractions, temperature, and pressure)
    * feed pump efficiency and pressure change (ΔP)
    * distillate HEX heat transfer coefficient, area, cold-side ΔP, and hot-side ΔP

>     * brine HEX heat transfer coefficient, area, cold-side ΔP, and hot-side ΔP

Reviewer: ```suggestion
    * brine HEX heat transfer coefficient, cold-side ΔP, and hot-side ΔP
```

>     * brine HEX heat transfer coefficient, area, cold-side ΔP, and hot-side ΔP

Reviewer: ```suggestion
    * brine HEX heat transfer coefficient, cold-side ΔP, and hot-side ΔP
```

>     * evaporator outlet brine temperature and overall heat transfer coefficient

Reviewer: ```suggestion
    * evaporator overall heat transfer coefficient
```

>     * compressor efficiency and pressure ratio

Reviewer: ```suggestion
    * compressor efficiency
```

    * brine pump efficiency and ΔP
    * distillate pump efficiency and ΔP
    * translator block outlet TDS concentration

> Flowsheet Specifications

Reviewer: The table needs to be updated based on the modification to the DOF list (i.e., removing the pressure ratio, etc.). Alternately, lower and upper bounds (if any) on pressure ratio and evaporator temperature could be added.

Author: Good catch - thanks!

Mechanical Vapor Compression
============================

Introduction
------------

Mechanical Vapor Compression (MVC) desalination is an evaporative, compressor-driven process where vapor generated
from saline feedwater is compressed to raise its temperature and pressure. This compressed vapor then condenses,
releasing latent heat to evaporate the feedwater, thus producing distillate without external heating or cooling.
The system primarily consists of an evaporator/condenser and a compressor, with optional heat exchangers for enhanced
heat integration, and may utilize external steam to initiate the process.

Implementation
--------------

Figure 1 illustrates the process flow diagram for a Mechanical Vapor Compression (MVC) desalination system.

> In this configuration, feedwater is pumped and equally divided between the cold inlets of a distillate heat

Reviewer 2: ```suggestion
In this configuration, feedwater is pumped and divided between the cold inlets of a distillate heat
```
The feed is not necessarily equally divided between the two heat exchangers because the split fraction (i.e. proportion of feed directed to the distillate heat exchanger) is set equal to the water recovery in a constraint.

    * `Water <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_
    * `Seawater <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_

Documentation for the costing relationships can be found below.
    * `WaterTAP Costing Package <https://watertap.readthedocs.io/en/latest/technical_reference/costing/watertap_costing.html>`_

The objective function is to minimize the levelized cost of water, which can be represented by the following equation
where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
:math:`f_{util}` represents the utilization factor:

    .. math::

        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}

Degrees of Freedom
------------------
The following variables are initially specified for simulating the MVC flowsheet (i.e., degrees of freedom = 0):
    * feed water conditions (mass flow, mass fractions, temperature, and pressure)
    * feed pump efficiency and pressure change (ΔP)

>     * separator split fraction

Reviewer 2: ```suggestion
```
I would remove the separator split fraction because it is not fixed for simulating the flowsheet, but an initial guess of the recovery is provided.

Mechanical Vapor Compression
============================

Introduction
------------

> Mechanical vapor compression (MVC) is an energy recovery system that recycles waste heat from working fluids (usually steam),
> reducing a facility's energy consumption and operational costs. The process works by increasing the
> temperature and pressure of waste heat with a compressor and then transferring the excess heat via heat exchangers (HEX), ultimately
> generating a concentrated brine stream and a distillate stream.

Reviewer: ```suggestion
Mechanical Vapor Compression (MVC) desalination is an evaporative, compressor-driven process where vapor generated from saline feedwater is compressed to raise its temperature and pressure. This compressed vapor then condenses, releasing latent heat to evaporate the feedwater, thus producing distillate without external heating or cooling. The system primarily consists of an evaporator/condenser and a compressor, with optional heat exchangers for enhanced heat integration, and may utilize external steam to initiate the process.
```

Implementation
--------------

> Figure 1 shows the process flow diagram for MVC where half (default value) of the influent water is pumped into the cold inlet
> of a distillate HEX, and the other half is pumped into the cold inlet of a brine HEX. After receiving heat from the HEXs,
> the cold side outlets are mixed and sent to the evaporator, which produces steam and brine streams.
> The brine is sent to the hot side inlet of the brine HEX, transferring some of its heat to the influent before leaving the system.
> Meanwhile, the steam generated by the evaporator is sent to a compressor, increasing the temperature and pressure
> of the steam. The amount of energy needed to compress the steam is relatively small compared to the energy saved by
> utilizing the recovered waste heat, making this process highly energy efficient. The compressed vapor is recycled back to the
> evaporator which helps to produce even more steam. Finally, this distillate is sent to the hot side inlet of the distillate
> HEX, transferring some of its heat to the influent before leaving the system. The flowsheet relies

Reviewer: ```suggestion
Figure 1 illustrates the process flow diagram for a Mechanical Vapor Compression (MVC) desalination system. In this configuration, feedwater is pumped and equally divided between the cold inlets of a distillate heat exchanger (HEX) and a brine HEX. The preheated feed exiting these heat exchangers combines and enters the evaporator, where it absorbs heat from the condensing steam and vaporizes, leaving behind concentrated brine. The generated vapor is then directed to a compressor, which elevates its temperature and pressure. This compressed vapor is recycled back to the evaporator, where it condenses and provides the necessary heat for evaporating the preheated feed. The hot concentrated brine and distillate pass through heat exchangers to preheat the incoming feedwater before exiting the system. The flowsheet relies on the following key assumptions:
```
